### DISCUSSION TITLE
Click events inside the story

### DISCUSSION THREAD

**Discussion by leonardolessa** [mohitk05/react-insta-stories#224] [Q&A]
Hi guys,
How can I have click events inside the story content?
I tried not displaying the div that contains the previous and forward elements but I wasn't able to trigger the events inside the story, it looks like the propagation is being stopped in the midway.

Thanks

**Comment by tobyonono** [comment]
What worked for me was wrapping the component inside a parent div and then setting the z-index of the children to 0 instead of the 999 that it is.

Something like:

`.parentContainer > div {
z-index: 0 !important;
}`

Annoying to have to use !important though.
